choreolib/src/main/java/choreo/auto/AutoRoutine.java

```diff
@@ -308,9 +308,9 @@ public Command cmd() {
    */
   public Command cmd(BooleanSupplier finishCondition) {
     return Commands.either(
-        Commands.run(this::poll)
+        Commands.startRun(this::poll, this::poll)
```

Comment:
Why is this change related to the change in autonomous checks? It's a good change but needs its own PR and explanation of why.